includes/definitions/discovery/windows.yaml

```diff
@@ -0,0 +1,28 @@
+mib: DHCP-MIB
+modules:
+    os:
+        sysDescr_regex: '/Hardware:\s([A-Za-z0-9]+( [A-Za-z0-9]+)+)/([A-Za-z0-9]+( [A-Za-z0-9]+)+)\s-\sSoftware: Windows Version\s[0-9]+\.[0-9]+\s\(([A-Za-z0-9]+( [A-Za-z0-9]+)+)\)/'
```

Comment:
This does not extract the fields. Need to use capture groups

Reply:
@Jellyfrog - Is this something you can help me with? The regex identifier is the only component of this pull request that I'm not 100% comfortable with.

Reply:
I've run out of time tonight but we can optimise this regex. I'll try and get an example tomorrow.